### PR TITLE
Update edit-step-as-json to handle the antiforgery token introduced in 3.12.0

### LINKED ISSUE #94: Edit step as json nolonger works in Octopus 3.13.9 due to anti-forgery token
With Octopus deploy 3.13.9 it appears that they added anti-forgery tokens to the forms. Unfortunately this seems to be interacting with the Edit step as JSON option. 

> Failed to save the step. A required anti-forgery token was not supplied or was invalid. If you are using a browser, please refresh your browser page, then sign out and sign back in to Octopus Deploy again. If you are using the API directly please consider using an API Key for authentication. If you are using a tool provided by Octopus Deploy please upgrade to the latest version of that tool and try again.

### PR DESCRIPTION
**Pull request by matt-richardson**
Fixes #94
 
Tested against 3.16.3 and against 3.11.0.

### CODE REVIEW

**Comment by davidroberts63** [comment]
Thanks @matt-richardson. I appreciate your patience in my greatly delayed response on this. Looking at it today.

File: src/3.0/background/edit-step-as-json-handler.js [hunk 1]
```diff
@@ -58,9 +61,11 @@ pygmy3_0.editStepAsJsonHandler = (function() {
 		getJsonResponse(url, handler);
 	}
 
-	function handleEditedStepAsJsonRequest(request, sendResponse, octopusRoot, getJsonResponse, handleEditedStepGetResponse, putJsonRequest) {
+	function handleEditedStepAsJsonRequest(request, sendResponse, octopusRoot, getJsonResponse, handleEditedStepGetResponse, putJsonRequest, antiForgeryToken) {
```
Comment (davidroberts63) [inline src/3.0/background/edit-step-as-json-handler.js]: Why the `antiForgeryToken` parameter here? Looks like you pull it from the `request` object on line 67.

Comment (matt-richardson) [reply src/3.0/background/edit-step-as-json-handler.js]: Missed your reply - oops.
That does look odd - I'll take a look...

Comment (matt-richardson) [reply src/3.0/background/edit-step-as-json-handler.js]: Okay, fix pushed. Think it must have been an early attempt that got left behind.

**Comment by davidroberts63** [comment]
Okay. I'll merge and publish this weekend.

**Pull request merged**